flex-linux-setup/flex_linux_setup/flex_setup.py

```
@@ -303,6 +303,36 @@ def set_app_versions_from_arguments(brgsp):
+    """
+    version = app_versions['NODE_MODULES_BRANCH'].replace('/', '-')
+
+    if version.startswith('v') or version == 'nightly':
+        release_tag = version
+    else:
+        release_tag = 'admin-ui-' + version
+
+    url = '{0}/{1}/admin-ui-{2}-built.tar.gz'.format(admin_ui_releases_base_url, release_tag, version)
+
+    try:
+        request.urlopen(request.Request(url, method='HEAD'), timeout=30)
```

Ruff (0.16.0)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)